pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -134,8 +134,7 @@ def get_coupled_variables(self, variables):
         elif SEI_option == "electron-migration limited":
             # Scott Marquis thesis (eq. 5.94)
             eta_inner = delta_phi - phase_param.U_inner
-            j_sei = phase_param.kappa_inner * eta_inner / L_sei_inner
-
+            j_sei = phase_param.kappa_inner * eta_inner / L_sei_inner * pybamm.EqualHeaviside(eta_inner, 0)
```

Comment:
I think it would be good to add parentheses on this expression or reorder the operands.

When I look at this I see a fraction
```
(kappa_inner * eta_inner)  /  (L_sei_inner * heaviside(eta_inner, 0))
```
while the python order of operations make it
```
(kappa_inner * eta_inner / L_sei_inner) * heaviside(eta_inner, 0)
```

So something like this would be clearer:
```suggestion
            j_sei = pybamm.EqualHeaviside(eta_inner, 0) * phase_param.kappa_inner * eta_inner / L_sei_inner
```